method/meringue/meringue.r

```diff
@@ -122,73 +129,63 @@ if ("selected" %in% colnames(positions)) {
         positions <- positions[as.logical(positions$selected), c(1, 2)]
   }
 
+# Load configuration
+k <- config$k
+alpha <- config$alpha
+beta <- config$beta
+n_pcs <- config$n_pcs
+n_pcs <- ifelse(is.null(opt$n_pcs), n_pcs, opt$n_pcs)
+filterDist <- config$filterDist
+
 # Seed
 seed <- opt$seed
 set.seed(seed)
 
-## Your code goes here
-k <- config$k
-alpha <- config$alpha
-beta <- config$beta
+# Preprocessing
+counts <- Matrix::t(Matrix::readMM(matrix_file))
+colnames(counts) <- rownames(positions)
+counts <- cleanCounts(counts = counts, 
+                      min.reads = 100, 
+                      min.lib.size = 100, 
```

Comment:
- `min.lib.size = 100` means that cells with fewer than 100 genes will be removed, but we might prefer to use centralized cell filtering (during the QC step)?
- Regarding the thresholds for `min.reads` and `min.lib.size`, the default setting for both is 1 as seen in [here]( https://github.com/JEFworks-Lab/MERINGUE/blob/master/R/process.R#L25). However, I've noticed that all the vignettes use 100. Should we keep both thresholds at 1 as default, or is there a reason to prefer the higher setting?

Reply:
Good points! 
Default settings would then only remove all zero rows/cols. So it would be basically no filtering. 
I think that would be fine, and then have the vignettes closer to how a person trying the tool would do it.